src/components/WalletIntegrationProvider/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connectBlockchain, initGnosis } from 'actions/blockchain'
+// import { getCurrentAccount } from 'api'
+import { getSelectedProvider } from 'selectors/blockchain'
+import { WALLET_PROVIDER } from 'integrations/constants'
+import Web3 from 'web3'
+import config from 'config'
+
+
+export default class WalletIntegrationProvider extends Component {
+
+  constructor(props) {
+    super(props)
+    const { integrations, store } = props
+    const initializers = Object.keys(integrations).map(integrationName => integrations[integrationName])
+
+
+    // Execute providers inizialization sequentially
+    const init = (funcs, _store) => {
```

Comment:
I think the linter doesn't like `_variable` style variables. Please call it something else to convey more meaning